src/block-components/helpers/borders/edit.js

```diff
@@ -40,10 +40,13 @@ export const BorderControls = props => {
 	const {
 		getAttribute,
 		getAttrName,
+		updateAttributes,
 	} = useAttributeEditHandlers( props.attrNameTemplate )
 
 	const borderTypeValue = getAttribute( 'borderType' ) || props.borderTypeValue
 
+	applyFilters( 'stackable.block-component.helpers', null, getAttribute, updateAttributes )
```

Comment:
The hook name `stackable.block-component.helpers` is too generic and feels that it belong to the entire `helpers` and not just the `border`

I suggest using `stackable.block-component.helpers.borders`

```suggestion
	applyFilters( 'stackable.block-component.helpers', null, getAttribute, updateAttributes )
```